cellxgene_schema_cli/cellxgene_schema/utils.py

```diff
@@ -54,6 +54,18 @@ def remove_deprecated_features(adata: ad.AnnData, deprecated: List[str]) -> ad.A
     return adata
 
 
+def remap_deprecated_features(*, adata: ad.AnnData, remapped_terms: Dict[str, str]) -> ad.AnnData:
```

Comment:
want to reconcile the function name with the second kwarg var name? 

i.e. `remap_deprecated_features` and `remapped_features`
OR
`remap_deprecated_terms` and `remapped_terms`

Unless there's an abstract reason (literally) for the names precisely as they already are